src/test/mappingHandlers.test.ts

```diff
@@ -1,4 +1,4 @@
-import { subqlTest } from "@subql/testing";
+// import { subqlTest } from "@subql/testing";
```

Comment:
Revert or remove?

Reply:
The mock of Subql provide a file that has the import and the code commented, so the linter once I apply it throw errors because one of the rules is not have unused imports, so I will add a comment to the file but will not remove it.

What you think @Olshansk ?